sdks/python/setup.py

```diff
@@ -166,6 +166,14 @@ def get_version():
     'google-cloud-bigtable>=0.31.1,<1.1.0',
 ]
 
+INTERACTIVE_BEAM = [
+    'facets-overview>=1.0.0,<2',
```

Comment:
Could you sort these dependencies?

Reply:
They are already sorted. Do you want me to also sort GCP_REQUIREMENTS?

Reply:
Nevermind. This looks good.